capa/ida/plugin/item.py

```diff
@@ -315,7 +315,7 @@ def __init__(
             ea = int(location)
             super(CapaExplorerFeatureItem, self).__init__(parent, [display, ea_to_hex(ea), details])
         else:
-            super(CapaExplorerFeatureItem, self).__init__(parent, [display, "global", details])
+            super(CapaExplorerFeatureItem, self).__init__(parent, [display, "", details])
```

Comment:
we will need to do some additional work to determine when `global` should be used in the `Address` column,